units/systemd-bsod.service.in

```diff
@@ -0,0 +1,21 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Displays emergency message full screen.
+Documentation=man:systemd-bsod(8)
+ConditionVirtualization=no
+DefaultDependencies=no
+Conflicts=shutdown.target
+Before=shutdown.target
+After=systemd-battery-check.service
+
+[Service]
+RemainAfterExit=yes
+ExecStart={{LIBEXECDIR}}/systemd-bsod --continuous
```

Comment:
The `systemd-bsod` binary is installed under `/usr/bin`, but you use `LIBEXECDIR` here, which means `/usr/lib/systemd`. So either you need to use just `systemd-bsod --continuous` here or change the binary target in `src/journal/meson.build`.

Reply:
isn't that what removing 'public' should do?

Reply:
ah ok, that explains it then

Reply:
> I don't think so. We just don't check the `--help` output for executables not marked as `public` afaik. To install it under `/usr/lib/systemd` you'd have to append it to the `libexec_template`, not `executable_template` in `src/journal/meson.build`.

@1awesomeJ please make that change too, in the same commit

Reply:
Ohh...sorry I'm just getting the notification for these messages.

Hello @mrc0mmand,
Thank you for stopping by sir.